### PR TITLE
Re-run gas estimation before sending tokens

### PR DESCRIPTION
**Pull request by brunobar79**
This would help preventing "out of gas" errors

### CODE REVIEW

**Review by osdnk — APPROVED** [review]

File: src/screens/SendSheet.js [hunk 1]
```diff
@@ -300,12 +300,30 @@ export default function SendSheet(props) {
     }
 
     let submitSuccess = false;
+    let updatedGasLimit = null;
+    // Attempt to update gas limit before sending ERC20 / ERC721
+    if (selected?.address !== 'eth') {
```
Comment (jinchung) [inline src/screens/SendSheet.js:305]: can we use ETH_ADDRESS from `@rainbow-me/references` for `eth`

**Review by jinchung — APPROVED** [review]
🌮

File: src/screens/SendSheet.js (hunk 1 above)
Comment (brunobar79) [reply src/screens/SendSheet.js:305]: YES!

**Review by skylarbarrera — APPROVED** [review]
lgtm 👌

**Pull request merged**